test/functional/cmdLineTests/jvmtitests/build.xml

```diff
@@ -63,19 +64,27 @@
 				<if>
 					<equals arg1="${JCL_VERSION}" arg2="latest"/>
 					<then>
-						<property name="addExports" value="--add-exports java.base/jdk.internal.module=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach=ALL-UNNAMED" />
+						<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+							<src path="${src}"/>
+							<src path="${src_latest}"/>
+							<exclude name="**/modularityTests/**"/>
+							<compilerarg line="--add-exports java.base/jdk.internal.module=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach=ALL-UNNAMED" />
+							<classpath>
+								<pathelement location="${asm.jar}" />
+							</classpath>
+						</javac>
 					</then>
 					<else>
-						<property name="addExports" value="--add-exports java.base/jdk.internal.module=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach=ALL-UNNAMED" />
+						<javac srcdir="${src}" destdir="${build}" debug="true" fork="true" executable="${compiler.javac}" includeAntRuntime="false" encoding="ISO-8859-1">
+							<src path="${src}"/>
+							<src path="${src_current}"/>
+							<compilerarg line="--add-exports java.base/jdk.internal.module=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach.target=ALL-UNNAMED --add-exports java.base/com.ibm.tools.attach=ALL-UNNAMED" />
+							<classpath>
+								<pathelement location="${asm.jar}" />
+							</classpath>
+						</javac>
```

Comment:
Is it possible to use variable to reduce code duplication?

Reply:
Yes,  I can add variable  'addExports' and update as <compilerarg line='${addExports}' />. Though it will not reduce  duplicate lines as those are  attributes of javac task.